atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -207,7 +207,7 @@ def run(self):
             self.log.debug("annotations: %r", annotations)
             raise
 
-        labels = self.make_labels()
+        labels = self.make_labels(metadata)
```

Comment:
For the Koji task ID, you should use get_build_json() and follow the 'metadata', 'labels', and 'koji-task-id' keys, converting the resulting value to an int, like koji_promote does.

For the Koji build ID, the koji_promote plugin (which has already run) returns that so why not just get it straight from `self.workflow.exit_results[PLUGIN_KEY_KOJI_PROMOTE]`?